update/.devcontainer/install-dev-tools.sh

```diff
@@ -15,7 +15,7 @@ sed -i '/nodocs/d' /etc/yum.conf
 
 # enable experimental repository with latest development packages
 sed -i'' -e 's^/release/^/experimental/^' /etc/yum.repos.d/pmm2-server.repo
-percona-release enable original testing
+percona-release enable pmm2-client testing
```

Comment:
This one won't work on RHEL 7 without an additional effort to make it work :(

Reply:
we don't develop on RHEL 7, so let's spend time on things that we might not use at all.

Reply:
ok, now I get you. let's just drop pmm-2.0.0 from CI matrix

Reply:
I have tested `percona-release enable pmm2-client testing` on all PMM Server versions up to 2.30.0 - I couldn't get it to work. Here is [the fix](percona-release enable pmm2-client testing) I hope.